proto/streamlit/proto/VegaLiteChart.proto

```diff
@@ -28,6 +28,9 @@ message VegaLiteChart {
   // specified using Vega-Lite's inline API.
   DataFrame data = 2;
 
+  // If True, will overwrite the chart width spec to fit to container.
+  bool use_container_width = 3;
```

Comment:
I would double check that nothing was using `3` before. If you can't bother check then set this to `5` :D